src/main/java/org/spongepowered/api/entity/living/Sittable.java

```diff
@@ -42,5 +42,4 @@
      * @param sitting Whether this entity is sitting
      */
     void setSitting(boolean sitting);
-
```

Comment:
We have a few changes to the original Google java style in our [contribution guidelines](https://docs.spongepowered.org/en/latest/devs/guidelines/#code-style).